charts/velero/values.yaml

```diff
@@ -327,9 +336,14 @@ restic:
   # SecurityContext to use for the Velero deployment. Optional.
   # Set fsGroup for `AWS IAM Roles for Service Accounts`
   # see more informations at: https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html
-  securityContext: {}
+  podSecurityContext:
+    runAsUser: 0
```

Comment:
Any reason why change it to `runAsUser: 0` (root user)?
From the Velero container image, by default, it runs as `nobody`.
https://github.com/vmware-tanzu/velero/blob/v1.6.0/Dockerfile#L60

Reply:
This is the `restic:` section of the values file. Restic did already run as root (and I think this is necessary to backup things), but I moved it from beeing hardcoded (see [restic-daemonset.yaml@Line47](https://github.com/vmware-tanzu/helm-charts/pull/221/files#diff-098ff3169e23f0fd8394cfaab72d002c19232c7f59d7e30321df2162df86aa81L47)) to the `values.yaml` where users are able to change this.